taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - toolchain
+
+transforms:
+    - rb_taskgraph.build:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    #index:
+    #    type: raptor
+    description: Build Reference Browser from source code.
+    treeherder:
+        kind: build
+        symbol: B
+        tier: 2
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 7200
+        chain-of-trust: true
+    run:
+        using: gradlew
+        use-caches: false
+    apk-artifact-template:
+        type: file
+        name: public/target.{abi}.apk
+        path: '/builds/worker/checkouts/src/app/build/outputs/apk/{variant}/{fileName}'
```

Comment:
A `taskgraph` note: one of the goals is to allow some devs to edit the YAML without needing to know the specifics about the transforms that interpret the YAML. However, in cases like this, it's not obvious what variables can be used in this string interpolation, or why `{variant}` can't be used for other properties.

I'm not sure how to solve this, but just a :thinking: